cmd/proxy-to-grafana/proxy-to-grafana.go

```diff
@@ -62,6 +62,12 @@ func main() {
 		Hostname: *hostname,
 	}
 
+	// TODO(bradfitz,maisem): move this to a method on tsnet.Server probably.
+	if err := ts.Start(); err != nil {
```

Comment:
you can drop the call to `ts.Start` and just call `ts.LocalClient`
```
localClient, err := ts.LocalClient(); err != nil {
   ...
}
```

Reply:
I could but I like the explicit Start.